src/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -84,7 +84,7 @@ namespace System.Reflection.Metadata
 {
     public readonly partial struct ArrayShape
     {
```

Comment:
@tmat this ref is updated now after your PR but I did refresh this to correctly mark these fields as object.